db/seeds.rb

```diff
@@ -6,23 +6,30 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+categories = Category.create( [ { title: "Apparel" },{ title: "Snowboards" } ] )
 
-apparel = Category.first.items.create([{name: "Hoody",
-                      description: "Keeps you slightly warm",
-                      price: 80},
+apparel = Category.first.items.create( [
+                  { name: "Hoody",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.